qa/acceptance-tests/src/test/java/io/camunda/it/auditlog/AuditLogAuthorizationsIT.java

```diff
@@ -0,0 +1,275 @@
+
+    startProcessInstance(adminClient, PROCESS_ID, TENANT_A, USE_REST_API);
+    startProcessInstance(adminClient, PROCESS_ID_A, TENANT_A, USE_REST_API);
+    startProcessInstance(adminClient, PROCESS_ID_B, TENANT_B, USE_REST_API);
+
+    // wait until all process instance creation records are exported and audit log entries are
+    // created
+    RecordingExporter.processInstanceCreationRecords()
+        .withIntent(ProcessInstanceCreationIntent.CREATED)
+        .limit(3L);
+    Awaitility.await("Audit log entries are created for the created process instances")
+        .ignoreExceptionsInstanceOf(ProblemException.class)
+        .untilAsserted(
+            () -> {
+              final var auditLogItems =
+                  adminClient
+                      .newAuditLogSearchRequest()
+                      .filter(
+                          fn ->
+                              fn.entityType(AuditLogEntityTypeEnum.PROCESS_INSTANCE)
+                                  .operationType(AuditLogOperationTypeEnum.CREATE))
+                      .send()
+                      .join();
+              assertThat(auditLogItems.items()).hasSize(3);
+            });
+  }
+
+  @Test
+  void shouldIsolateAuditLogsByTenant(
+      @Authenticated(USER_A_USERNAME) final CamundaClient userAClient,
+      @Authenticated(USER_B_USERNAME) final CamundaClient userBClient) {
+    // when
+    final var userAAuditLogs = userAClient.newAuditLogSearchRequest().send().join();
+    final var userBAuditLogs = userBClient.newAuditLogSearchRequest().send().join();
+
+    // then
+    // User A should only see audit logs from TENANT_A and PROCESS_ID_A
+    assertThat(userAAuditLogs.items()).isNotEmpty();
+    assertThat(userAAuditLogs.items().stream().allMatch(log -> TENANT_A.equals(log.getTenantId())))
+        .isTrue();
+    assertThat(
+            userAAuditLogs.items().stream()
+                .anyMatch(log -> PROCESS_ID_A.equals(log.getProcessDefinitionId())))
+        .isTrue();
+    assertThat(
+            userAAuditLogs.items().stream()
+                .noneMatch(log -> PROCESS_ID_B.equals(log.getProcessDefinitionId())))
+        .isTrue();
+
+    // User B should only see audit logs from TENANT_B and PROCESS_ID_B
+    assertThat(userBAuditLogs.items()).isNotEmpty();
+    assertThat(userBAuditLogs.items().stream().allMatch(log -> TENANT_B.equals(log.getTenantId())))
+        .isTrue();
+    assertThat(
+            userBAuditLogs.items().stream()
+                .anyMatch(log -> PROCESS_ID_B.equals(log.getProcessDefinitionId())))
+        .isTrue();
+    assertThat(
+            userBAuditLogs.items().stream()
+                .noneMatch(log -> PROCESS_ID_A.equals(log.getProcessDefinitionId())))
+        .isTrue();
+  }
+
+  @Test
+  void shouldNotSeeDeployedResourceAuditLogsWithOnlyUserTasksCategoryPermission(
+      @Authenticated(USER_C_USERNAME) final CamundaClient userCClient) {
+    // when
+    final var userCAuditLogs =
+        userCClient
```

Comment:
🔧 how about naming this something like `userTaskUserClient` or so? A/B/C is pretty generic 😄